GitUI/CommandsDialogs/FormPull.cs

```diff
@@ -1163,6 +1164,17 @@ private void PruneTags_CheckedChanged(object sender, EventArgs e)
             AllTags.Checked = AllTags.Checked || PruneTags.Checked;
         }
 
+        private void DisableMergeOptionsWhenCurrentBranchIsLocal()
+        {
+            var isLocalBranch = string.IsNullOrEmpty(Module.GetRemoteBranch(localBranch.Text));
+
+            Merge.Checked = !isLocalBranch;
+            Merge.Enabled = !isLocalBranch;
+            Rebase.Enabled = !isLocalBranch;
+            Fetch.Checked = !isLocalBranch;
+            Fetch.Enabled = !isLocalBranch;
```

Comment:
[nit] we can simplify this a little bit by chaining assignments:
```suggestion
            Rebase.Enabled =
                Merge.Checked =
                Merge.Enabled =
                Fetch.Checked =
                Fetch.Enabled = !isLocalBranch;
```